main.go

```diff
@@ -134,18 +135,45 @@ func run(_ context.Context) error {
 			}
 
 			m[g.ID] = guildConfig
+
+			request, err := session.Request(http.MethodGet, fmt.Sprintf("%s/%s", discordgo.EndpointGuild(g.ID), "soundboard-sounds"), nil)
+			if err != nil {
+				logger.Error("could not sent message", slog.String("err", err.Error()))
+			} else {
+				logger.Debug("sounds:" + string(request))
+			}
 		}
 	})
-
 	session.AddHandler(func(s *discordgo.Session, vs *discordgo.VoiceStateUpdate) {
-		logger = logger.With(slog.String("username", vs.Member.User.Username), slog.String("guild", vs.GuildID), slog.String("channel", vs.ChannelID))
+		logger := logger.With(slog.String("username", vs.Member.User.Username), slog.String("guild", vs.GuildID), slog.String("channel", vs.ChannelID))
 
 		logger.Info("joined")
 		c, ok := m[vs.GuildID]
 		if !ok {
 			logger.Warn("unknown guild")
 			return
 		}
+		logger.Info("joined", vs.Member.User.Username)
+		if vs.Member.User.Username == "jcalyon" {
```

Comment:
TODO:
Add a slash command to let callers register a sound to auto play when they join voice
Move this particular config to the config file so that the code is more portable